the_debuginator.h

```diff
@@ -2377,6 +2377,30 @@ void debuginator_move_to_parent(TheDebuginator* debuginator) {
 	}
 }
 
+void debuginator_move_to_root(TheDebuginator* debuginator) {
+	DebuginatorItem* hot_item = debuginator->hot_item;
+	while (hot_item && hot_item != debuginator->root) {
```

Comment:
I'm confused :)
1) In the current Debuginator, folders can't be expanded/collapsed (and leaves can't have sub-leaves).
2) If you are in a leaf, all folders above it (once my folder-collapse-feature is checked in) will already be opened. Unless you mean things like the leaf's parent's previous sibling?
3) When you expand a child all it's children are collapsed?